Bio/PDB/QCPSuperimposer/__init__.py

```diff
@@ -2,8 +2,7 @@
 # This code is part of the Biopython distribution and governed by its
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
-"""
-QCPSuperimposer finds the best rotation and translation to put
+"""QCPSuperimposer finds the best rotation and translation to put
```

Comment:
Leave the main text as it was, insert a one line summary like ``Structural alignment using Quaternion Characteristic Polynomial (QCP).``